samples/apps/copilot-chat-app/webapi/appsettings.json

```diff
@@ -151,37 +151,17 @@
   },
 
   //
-  // ChatSkill prompts are used to generate responses to user messages.
+  // Prompts are used to generate responses to user messages.
   // - CompletionTokenLimit is the token limit of the chat model, see https://platform.openai.com/docs/models/overview
   //   and adjust the limit according to the completion model you select.
-  // - ResponseTokenLimit is the token count left for the model to generate text after the prompt.
   //
   "Prompts": {
     "CompletionTokenLimit": 4096,
```

Comment:
I don't think we need this now that we have the config files.

Reply:
We need it for the chat history skill. That doesnt have a prompt.txt or config.json to configure its token limits.

Reply:
Since this only applies to native functions now, can we rename this to maybe "NativeSkillSettings"?

Reply:
This is also used by Semantic Skills. Both KnowledgeCutoffDate and MemoryFormat are.